source/neuropod/bindings/java/README.md

```diff
@@ -1,42 +1,41 @@
-# The java api interface:
+# Java API
 
-1. To build & to unit test:
-```
-bazel build //neuropod/bindings/java:neuropod_java_jar
-bazel test //neuropod/bindings/java:*
```

Comment:
As of #413, you need to use `./build/test.sh` or `./build/run_cpp_tests.py` to successfully run the java tests

Reply:
I think we can specify that build and test above can be used only during development (for smoke test). But then need to perform ./build/test.sh or ./build/run_cpp_tests.py

Reply:
The test command in the README will not pass anymore as of #413 without running under either of the scripts I mentioned (this is because some of the java tests now require env variables to be set by the test runner).

I'll change some of our build tools to make it easier for us to run tests for a subset of the project